test/cli/checkers-report_test.py

```diff
@@ -0,0 +1,17 @@
+# python -m pytest checkers-report_test.py
+
+import os
+import xml.etree.ElementTree as ET
+
+from testutils import cppcheck
+
+__script_dir = os.path.dirname(os.path.abspath(__file__))
+__project_dir = os.path.join(__script_dir, 'checkers-testing')
+
+def test_xml_checkers_report():
```

Comment:
I rather group tests together a bit. at least there should be 2-3 tests in a file.